docs/api/rest/index.rst

```diff
@@ -40,7 +40,7 @@ If all you want is reference guides, skip straight to :ref:`rest-api-schemas`.
 
 .. versionchanged:: 3.1
 
-   The API version was bumped to v1.3 in Patchwork v3.1. The older APIs are
+   The API version was bumped to v1.4 in Patchwork v3.1. The older APIs are
    still supported. For more information, refer to :ref:`rest-api-versions`.
```

Comment:
v3.1 is already out, so you'll want to add a new directive indicating that we bumped to API v1.4 in Patchwork v3.2

Reply:
Fixed! About the API / Application versioning considering my other PR #441 can they both be on v1.4 / v3.2 or each PR needs a separate bump in versions ?